apps/blockchain-mailer/src/klass/ContractWatcher.ts

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
You mean the `klass` folder? I use this name because I want avoid `class`. I'm not sure what name would be better if we want group a lot of classes in a folder.

Reply:
Right, our projects usually group files by functionality rather than type of file. You can leave it as-is for now and I will think about the architecture more.

An example is in our NestJS projects, where all `user` functionality will be grouped in a folder called `user/`, with files such as `user.controller.ts`, `user.service.ts`, `user.model.ts`, `user.interface.ts`, etc. That doesn't quite fit a service like this, but we can adopt something similar.

Reply:
`our projects usually group files by functionality rather than type of file` 
Got it! I'll refactor the structure to this way.

Reply:
@solomondefi I'll pending this (or move to another issue) because I can't find a good structure right now. Please consider merge this PR first.

Reply:
That's fine, no problem.